src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts

```diff
@@ -76,10 +76,13 @@ export class HuiDialogEditLovelace extends LitElement {
             ?disabled="${!this._config || this._saving}"
             @click="${this._save}"
           >
-            <ha-circular-progress
-              ?active="${this._saving}"
-              alt="Saving"
-            ></ha-circular-progress>
+            ${this._saving
+              ? html`<ha-circular-progress
+                  active
+                  size="small"
+                  title="Saving"
+                ></ha-circular-progress>`
+              : ""}
```

Comment:
Should not the spinner replace the text when active?
![image](https://user-images.githubusercontent.com/15093472/95462917-70e95f00-0978-11eb-8fe7-fd0b83bcb3e0.png)

Reply:
I intentionally didn't change the current behavior... But we should align it some time I think. We use different ways at different places

Reply:
Oh... right, that's not related to this change, just noticed while testing it :D